jsonnet/kube-prometheus/components/prometheus.libsonnet

```diff
@@ -98,6 +98,7 @@ function(params) {
     apiVersion: 'v1',
     kind: 'ServiceAccount',
     metadata: p._metadata,
+    automountServiceAccountToken: false,
```

Comment:
This part can be treated as a breaking change for folks who are using prometheus-operator in versions that do not have change from https://github.com/prometheus-operator/prometheus-operator/pull/4514. Do you think we should wait with this until we have a new operator version and then circle back?

Reply:
yes sounds good

Reply:
Just out of curiosity, what is the next release date?

Reply:
Yesterday :smile: